backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Controllers/LessonsController.cs

```diff
@@ -24,6 +24,13 @@ public async Task<ActionResult<ICollection<LessonDto>>> GetAllAsync([FromBody] F
             return Ok(lessons);
         }
 
+        [HttpGet("week")]
+        public async Task<ActionResult<ICollection<DayCardDto>>> GetDayCardAsync([FromQuery] RequestDayCardDto requestDto)
+        {
+            var lessons = await _lessonsService.GetDayCardsOfWeekAsync(requestDto);
+            return Ok(lessons);
+        }
```

Comment:
```suggestion
        public async Task<ActionResult<ICollection<DayCardDto>>> GetDayCardAsync([FromQuery] RequestDayCardDto requestDto) => _lessonsService.GetDayCardsOfWeekAsync(requestDto);
```

Reply:
This change requires also method signature change like
`[HttpGet("week")]
        public async Task<ICollection<DayCardDto>> GetDayCardAsync([FromQuery] RequestDayCardDto requestDto)
            => (await _lessonsService.GetDayCardsOfWeekAsync(requestDto))!;`
Should we change all the methods in controller in the same way?

Reply:
fixed